tests/libtest/lib500.c

```diff
@@ -96,36 +96,40 @@ int test(char *URL)
     if(libtest_arg2) {
       FILE *moo = fopen(libtest_arg2, "wb");
       if(moo) {
-        double time_namelookup;
-        double time_connect;
-        double time_pretransfer;
-        double time_starttransfer;
-        double time_total;
+        curl_off_t time_namelookup;
+        curl_off_t time_connect;
+        curl_off_t time_pretransfer;
+        curl_off_t time_starttransfer;
+        curl_off_t time_total;
         fprintf(moo, "IP: %s\n", ipstr);
-        curl_easy_getinfo(curl, CURLINFO_NAMELOOKUP_TIME, &time_namelookup);
-        curl_easy_getinfo(curl, CURLINFO_CONNECT_TIME, &time_connect);
-        curl_easy_getinfo(curl, CURLINFO_PRETRANSFER_TIME, &time_pretransfer);
-        curl_easy_getinfo(curl, CURLINFO_STARTTRANSFER_TIME,
+        curl_easy_getinfo(curl, CURLINFO_NAMELOOKUP_TIME_T, &time_namelookup);
```

Comment:
Since you limited the code to extract this time to 64 bit only, won't this code fail hard on 32 bit systems?

Reply:
Made it 32-bit also.